dynamic-workspace/dynamic-workspace.qml

```diff
@@ -0,0 +1,43 @@
+import QtQml 2.0
+import QOwnNotesTypes 1.0
+import com.qownnotes.noteapi 1.0
+
+/**
+ * This script exports multiple notes as one HTML file
```

Comment:
@fnkabit, the description is not very fitting. 😅

Reply:
I fixed that now in https://github.com/qownnotes/scripts/commit/a73ff4b00d5217cef526387f3101cabcb635f8ba